### ISSUE TITLE
Extension relation for origin cell of component

### ISSUE THREAD

**Issue by rachhuntley** [geneontology/annotation_extensions#48]
Hi,

Is there a relation in RO that would be applicable to make this kind of annotation;

Gene product X is located in (GO:0070062) extracellular exosome that has derived from a cardiac fibroblast cell.

I thought there was a relation derives_from, but it doesn't exist in RO. There is a relation produced_by, would this work?

If so, could you make it valid for use in annotation extensions?

UniProt have made many such annotations for their exosome annotation project, but they are currently using part_of, which I think is wrong. I would also like to make this type of annotation for exosomal location of miRNAs.

Thanks,
Rachael.

**Comment by cmungall** [comment]
If the exosome is a product of the fibroblast then yes, produced_by

Does fibroblast reflect the in-vivo biology or the cell line?

**Comment by rachhuntley** [comment]
Yes, it's in vivo. The cardiac fibroblasts were isolated from rats and cultured, they purified exosomes from the conditioned medium of the fibroblasts.

Could you please do the necessary edits to allow produced_by to be used in annotation extensions?
This would include adding subsets, which I'm assuming would be the following (maybe more? There is no definition for produced_by);
subset: AE_cell_or_anatomical
subset: AE_cellular_component
subset: display_for_curators

Thanks,
Rachael.

**Comment by dosumis** [comment]
I'll add ‘display_for_curators and AE_cellular_component.

I don’t think "subset: AE_cell_or_anatomical” is appropriate as in anatomy/cell ontologies we would use develops_from where there is a lineage relationship.

Possible def (semi formal):
B produced_by A iif:
A exists at time t1
A and B exist at time t2
A and B are distinct entities (B not overlaps A)
t2 preceded_by t1
B consists largely or completely of material that part_of A at t1.

(Note - this def would cover budding in cell lineage, but for consistency, develops_from should still be used for this until/unless we have a budding relatinoship It would also cover secretion.)

Needs a comment to avoid usage for lineage:

**comment:** For cell lineage relationships, use develops_from or its descendent relations/

Usage comment for annotation extension needs to be more restricted. For now how about:

**usage:** Use this relation to link an extracellular cellular_component that is a product of a cell, to a cell that produced it.  Examples include exosomes, LDLs, secreted ECM components.
**local_domain:** cellular_component
**local_range:** cell

We could always extend it in future. Another possible usage might be recording the origin of CCs from other CCs.  Can't think of good examples right now though.

**Comment by rachhuntley** [comment]
Hi David,

Thanks for this. How do we go about making a new GitHub page for this relation? I'm sure you've mentioned this before but I can't find it amongst all the emails.

Rachael.

**Comment by dosumis** [comment]
> Thanks for this. How do we go about making a new GitHub page for this relation?

Will be created autoautomatically when I run the update script.  Will do so ASAP and let you know.

**Comment by dosumis** [comment]
Just realised that, whereas produced by lacks a definition, its inverse, produces, has one:

label "produces"@en
comment "Note that this definition doesn't quite distinguish the output of a transformation process from a production process, which is related to the identity/granularity issue."^^string
definition "a produces b if some process that occurs_in a has_output b, where a and b are material entities. Examples: hybridoma cell line produces monoclonal antibody reagent; chondroblast produces avascular GAG-rich matrix."^^string
'has curation status' 'pending final vetting'
in_subset 'eco subset'
'term editor' "Melissa Haendel"

This is reaonable, but makes it a more general relation than my proposed def, as it does not require secretion.  With this definition, produced_by would be a superclass to expressed_by. @mellybelly can you comment on usage?  Could the def be tightened?

(Note, discussion should really be on  RO tracker.  Will move it there once I have permissions).

**Comment by dosumis** [comment]
Now has page under doc on this site.  No def yet though - just a usage statement.  Probably best to hold off on use until this is settled.
